src/main/java/tictac7x/charges/items/jewelry/J_EfaritaysAid.java

```diff
@@ -24,49 +27,42 @@ public J_EfaritaysAid(final Provider provider) {
 
             // Charges from break dialog.
             new OnWidgetLoaded(219, 1, 0).text("Status: (?<charges>.+) charges? left.").setDynamically().onMenuOption("Break").onMenuTarget("Efaritay's aid"),
 
             // Attack tier-2 vampyre.
-            new OnHitsplatApplied(HitsplatTarget.ENEMY).hasTargetName("Vampyre Juvinate").isEquipped().multiTrigger().decreaseCharges(1),
+            new OnHitsplatApplied(HitsplatTarget.ENEMY, HitsplatGroup.ALL).hasTargetName(
+                // Tier 1
+                "Count Draynor",
+                "Dessous",
+                "Feral Vampyre",
+                "Kroy",
+                "Vampyre Juvenile",
 
-            // Additional charge used with silver melee weapon on successful hit.
-            new OnHitsplatApplied(HitsplatTarget.ENEMY, HitsplatGroup.ALL).hasTargetName("Vampyre Juvinate").combatStyle(CombatStyle.MELEE).isEquipped().multiTrigger().itemEquipped(
-                ItemId.BLESSED_AXE,
-                ItemId.IVANDIS_FLAIL,
-                ItemId.BLISTERWOOD_FLAIL,
-                ItemId.SILVER_SICKLE,
-                ItemId.SILVER_SICKLE_BLESSED,
-                ItemId.EMERALD_SICKLE_BLESSED,
-                ItemId.EMERALD_SICKLE_BLESSED_ENCHANTED,
-                ItemId.RUBY_SICKLE_BLESSED,
-                ItemId.RUBY_SICKLE_BLESSED_ENCHANTED,
-                ItemId.BLISTERWOOD_SICKLE,
-                ItemId.SILVERLIGHT,
-                ItemId.DARKLIGHT,
-                ItemId.ARCLIGHT,
-                ItemId.ARCLIGHT_UNCHARGED,
-                ItemId.EMBERLIGHT,
-                ItemId.ROD_OF_IVANDIS_1,
-                ItemId.ROD_OF_IVANDIS_2,
-                ItemId.ROD_OF_IVANDIS_3,
-                ItemId.ROD_OF_IVANDIS_4,
-                ItemId.ROD_OF_IVANDIS_5,
-                ItemId.ROD_OF_IVANDIS_6,
-                ItemId.ROD_OF_IVANDIS_7,
-                ItemId.ROD_OF_IVANDIS_8,
-                ItemId.ROD_OF_IVANDIS_9,
-                ItemId.ROD_OF_IVANDIS_10,
-                ItemId.WOLFBANE
-            ).decreaseCharges(1),
+                // Tier 2
+                "Vampyre Juvinate",
 
-            // Additional charge used with silver bolts on successful hit.
-            new OnHitsplatApplied(HitsplatTarget.ENEMY, HitsplatGroup.ALL).hasTargetName("Vampyre Juvinate").combatStyle(CombatStyle.RANGED).isEquipped().multiTrigger().itemEquipped(
-                ItemId.SILVER_BOLTS
-            ).decreaseCharges(1),
+                // Tier 3
+                "Damien Leucurte",
+                "Ranis Drakan",
+                "Vanstrom Klause",
+                "Vyrewatch Sentinel",
+                "Vyrewatch"
+            ).isEquipped().decreaseCharges(1).consumer(() -> {
+                attackedVampyre = true;
+            }),
+            new OnAnimationChanged(AnimationId.THRALL_SKELETON, AnimationId.THRALL_GHOST, AnimationId.THRALL_ZOMBIE).actorName("null").isEquipped().consumer(() -> {
```

Comment:
Does this attackedVampyre and thrall interaction do anything or is that just a leftover code?